src/options.rs

```diff
@@ -93,12 +93,12 @@ impl Options {
             depth: if matches.is_present("recursive") {
                 None
             } else {
-                let i = matches.value_of("depth").map(|s| s.parse::<isize>().unwrap()).unwrap_or(0);
+                let i = matches.value_of("depth").map(|s| s.parse::<isize>().unwrap()).unwrap_or_default();
                 if i < 0 { None } else { Some(i as usize) }
             },
             file: file,
             follow_symlinks: !matches.is_present("no-follow-symlinks"),
-            ignored_files: matches.values_of("ignore").map(|v| v.map(String::from).collect()).unwrap_or(BTreeSet::new()),
+            ignored_files: matches.values_of("ignore").map(|v| v.map(String::from).collect()).unwrap_or_default(),
```

Comment:
`Option::unwrap_or(*::new)` is considered an anti-pattern (mostly because it involves creating a new value, even if the option is Some). Would you be okay with `.map_or_else(BTreeSet::new, |v| v.map(String::from).collect())` (which is usually preferred over `.map(|v|...).unwrap_or_else(BTreeSet::new)`)